scripts/generateDocs.js

```diff
@@ -47,8 +47,8 @@ function formatTemplateFileContents(content, destination) {
    * Replace placeholder strings
    */
   result = result.replace(/\[npm]/g, nameExtractionData.npm);
-  result = result.replace(/\[name]/g, nameExtractionData.name);
-  result = result.replace(/\[Name]/g, nameExtractionData.nameCap);
+  result = result.replace(/\[name](?!\()/g, nameExtractionData.name);
+  result = result.replace(/\[Name](?!\()/g, nameExtractionData.nameCap);
```

Comment:
What is the reason for this change? How does this change the output of the generated docs? Do we need to make this update to the Auro Generator? Or did this come from the Generator?

Reply:
This update came from the Generator